aip/aog/3001.md

```diff
@@ -7,9 +7,9 @@ aip:
   type: process
   scope: aog
 js:
-  - /assets/js/graphviz/viz.js
-  - /assets/js/graphviz/lite.render.js
-  - /assets/js/aip/aip-graphviz.js
+  - /static/js/graphviz/viz.js
+  - /static/js/graphviz/lite.render.js
+  - /static/js/aip/aip-graphviz.js
```

Comment:
I'm assuming there is some tooling that uses `static` instead of `assets` by default. But if the tooling is configurable, then keeping this as `assets` would reduce the migration cost some.

Even better if the tooling can be updated to support either or both.

Reply:
You are correct, [Flask likes "static"](https://flask.palletsprojects.com/en/1.1.x/quickstart/#static-files) for this purpose. Surprisingly, I see no obvious [configuration](https://flask.palletsprojects.com/en/1.1.x/config/#configuration-basics) to use something else. I agree with preferring "assets" to remove this migration point, but assert the importance is low.